<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainFrame</class>
 <widget class="QWidget" name="mainFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1437</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="te_output">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>30</y>
     <width>481</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>891</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>DB name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_db_name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Table Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_table_name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Ref.Col.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_col_name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>DataFile</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_data_file_name">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>2</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Upload Option:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_ignore">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Ignore</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_replace">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Replace</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Download</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_4">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Upload</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_5">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Refres</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cb_db_name</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>mainFrame</receiver>
   <slot>set_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>565</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>mainFrame</receiver>
   <slot>download_from_server()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>425</y>
    </hint>
    <hint type="destinationlabel">
     <x>-30</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>mainFrame</receiver>
   <slot>connect_to_db()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>492</y>
    </hint>
    <hint type="destinationlabel">
     <x>-37</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_table_name</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>mainFrame</receiver>
   <slot>on_cb_table_name_changed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>-25</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_ignore</sender>
   <signal>clicked()</signal>
   <receiver>mainFrame</receiver>
   <slot>set_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rb_replace</sender>
   <signal>clicked()</signal>
   <receiver>mainFrame</receiver>
   <slot>set_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>687</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>710</x>
     <y>559</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>mainFrame</receiver>
   <slot>write_to_server()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>653</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>808</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_col_name</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>mainFrame</receiver>
   <slot>set_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>-83</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_data_file_name</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>mainFrame</receiver>
   <slot>set_params()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>-46</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>mainFrame</receiver>
   <slot>refresh_data_foler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>874</x>
     <y>450</y>
    </hint>
    <hint type="destinationlabel">
     <x>874</x>
     <y>588</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>set_params()</slot>
  <slot>download_from_server()</slot>
  <slot>init_gui()</slot>
  <slot>connect_to_db()</slot>
  <slot>on_cb_table_name_changed()</slot>
  <slot>write_to_server()</slot>
  <slot>refresh_data_foler()</slot>
 </slots>
</ui>
